features/remotepollerng/daemon/src/main/java/org/opennms/netmgt/remotepollerng/RemotePollerd.java

```diff
@@ -321,25 +327,52 @@ public void saveResponseTimeData(final String locationName, final OnmsMonitoredS
             return;
         }
 
-        // FIXME: We don't need to recompute this everytime
-        RrdRepository repository = new RrdRepository();
-        repository.setStep(pollerConfig.getStep(pkg));
-        repository.setHeartBeat(repository.getHeartBeat());
-        repository.setRraList(pollerConfig.getRRAList(pkg));
+        // TODO: Apply thresholding
```

Comment:
Should be easy to apply thresholding now that we have the `CollectionSet`.

Reply:
There is a issue for that: https://issues.opennms.org/browse/NMS-12721